src/Microsoft.NetCore.Analyzers/UnitTests/Security/SslProtocolsAnalyzerTests.cs

```diff
@@ -478,14 +479,14 @@ public void Method()
         SslProtocols protocols = SslProtocols.Tls12 | (SslProtocols)192;
     }
 }",
-                GetCSharpResultAt(9, 34, SslProtocolsAnalyzer.HardcodedRule, "Tls12"),
-                GetCSharpResultAt(9, 34, SslProtocolsAnalyzer.DeprecatedRule, "3264"));
+                VerifyCS.Diagnostic(SslProtocolsAnalyzer.DeprecatedRule).WithSpan(9, 34, 9, 52).WithArguments("3264"),
```

Comment:
Test is failing saying the other expected diagnostic appears first. When I change the order, it's then the other test appearing first.

Reply:
@sharwell ?

Reply:
@mavasani I could keep the base class and the old test verification for this specific test and create a ticket for this issue. Otherwise I will need some help for this case.

Reply:
Sounds good. Anything that helps you move forward with the PR works for me. Thanks!

Reply:
See #3091